src/orbital/azext_orbital/aaz/latest/orbital/contact_profile/_create.py

```diff
@@ -0,0 +1,550 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "orbital contact-profile create",
+)
+class Create(AAZCommand):
+    """Creates or updates a contact profile.
```

Comment:
```suggestion
    """Create a contact profile.
```

Reply:
yeah the PUT API behind this cmdlet does both create and update.

Reply:
All the summaries are from our public docs https://docs.microsoft.com/en-us/rest/api/orbital/earthobservation/contact-profiles/create-or-update?tabs=HTTP